python/lsst/meas/base/plugins.py

```diff
@@ -68,6 +71,103 @@
                     TransformClass=bl.ApertureFluxTransform, executionOrder=BasePlugin.FLUX_ORDER)
 
 # --- Single-Frame Measurement Plugins ---
+class SingleFrameVarianceConfig(SingleFramePluginConfig):
+        scale = lsst.pex.config.Field(dtype=float, default=5.0, optional=True,
+                                      doc="scale factor to apply to shape for aperture")
+        mask = lsst.pex.config.ListField(doc="mask planes to ignore", dtype=str,
+                                         default=["DETECTED", "DETECTED_NEGATIVE"])
```

Comment:
I simply ported the existing behavior, It probably is a good idea to have them all masked. I will make the change unless you can think of some reason not to now.